easybuild/tools/filetools.py

```diff
@@ -1158,3 +1102,61 @@ def det_size(path):
         _log.warn("Could not determine install size: %s" % err)
 
     return installsize
+
+
+def copy_files(src, dst, symlinks=False, ignore=None):
+    """
+    Copy the list of files or directories to a destintation
+    @src list or string of files and directories to copy
+    @dst the destination directory
+    @symlinks copy symlinks as symlinks
+    @ignore files to ignore in case the src is a directory
+    """
+    if build_option('extended_dry_run'):
+        dry_run_msg("Copying directory %s to %s" % (src, dst), silent=build_option('silent'))
+        return
+
+    if os.path.exists(dst):
+        if not os.path.isdir(dst):
+            raise EasyBuildError("File with the same name as destination directory %s \
+                                  already exists.", dst)
+        elif not os.path.listdir(dst):
+            raise EasyBuildError("Destination directory %s of copy already exists.", dst)
+        else:
+            try:
+                _log.debug("Removing empty directory %s." % dst)
+                os.rmdir(dst)
+            except OSError, err:
+                raise EasyBuildError("Failed to remove %s: %s", dst, err)
+
+    if isinstance(src, basestring):
+        src = [src]
+
+    try:
+        for item in src:
+            if os.path.isfile(item):
+                _log.debug("Copying file %s to %s" % (item, dst))
+                shutil.copy2(item, dst)
```

Comment:
you should take into account that individual files can also be symlinks, since `copy2` resolves the symlink rather than copying it

cfr. http://stackoverflow.com/questions/4847615/copying-a-symbolic-link-in-python